pkgs/applications/audio/pianoteq/default.nix

```diff
@@ -0,0 +1,200 @@
+{ lib, stdenv, curl, gnugrep, jq, xorg, alsaLib, freetype, p7zip, autoPatchelfHook, writeShellScript, zlib, libjack2, makeWrapper }:
```

Comment:
```suggestion
{ lib, stdenv, curl, gnugrep, jq, xorg, alsa-lib, freetype, p7zip, autoPatchelfHook, writeShellScript, zlib, libjack2, makeWrapper }:
```

Reply:
applied in d040aff89639da4383473cb7afdd3e98d579b164